### DISCUSSION TITLE
Unable to Create a Open AI Service in Azure

### DISCUSSION THREAD

**Discussion by Koushik1984** [AzureAiDevs/hub#11] [azure-ai-developers]
Unable to Create an Open AI Service in Azure for 30-Day Challenge. It's requesting a Form to be completed and take 10 business days

**Comment by revodavid** [comment]
Yes, you do need to complete that form to access Azure OpenAI service. I've heard that for most the response time is much quicker than 10- days though.

**Reply by leballard** [reply]
If I am trying to learn Azure AI on my own, how do I access Azure OpenAI?  The form says only business accounts will be created
